src/main/java/org/molgenis/vcf/decisiontree/filter/model/FieldType.java

```diff
@@ -4,8 +4,13 @@ public enum FieldType {
   COMMON,
   INFO,
   /**
-   * INFO field with nested information (e.g. VEP CSQ or SnpEff ANN)
+   * INFO field with nested information (VEP CSQ)
    */
   INFO_VEP,
-  FORMAT
+  FORMAT,
+  /**
+   * FORMAT field with nested information (GENOTYPE info from htsjdk)
+   */
+  FORMAT_GENOTYPE,
+  SAMPLE
```

Comment:
link to enum with allowed values

Reply:
outdated: fix is on another line so not recognized by github